pending/exports-and-boilerplate.pod

```diff
@@ -0,0 +1,116 @@
+=encoding utf8
+
+=head1 Reducing boilerplate and managing exports in Dancer2
+
+Dancer makes it easy to start small and grow fast. Today we will be looking at a techique to help manage that growth.
+
+A typical mature Dancer2 app will span multiple packages and each package will make use of Dancer's plugins and other modules available on CPAN.
+
+  use Scalar::Util qw< blessed >;
+  use List::Util qw< first all none sum >;
+  use Dancer2 appname => MyApp;
+  use Dancer2::Plugin::DBIC qw< schema resultset >;
+  use Dancer2::Plugin::Redis;
+
+After the first couple of files, the copy-pasting becomes tedious and if you later change your mind about what to include, you leave yourself open to runtime errors when you try to call a function you never imported into the package namespace.
+
+=head2 What Dancer2 "exports"
+
+Dancer2 "exports" almost eighty functions into the symbol table by default. That's a lot! You can get a complete list of them from the manual or from the symbol table itself with this one-liner:
+
+  perl -le 'use Dancer2; print for grep { main->can($_) && s/^\*main:://} map {$::{$_}} sort keys %main::;'
+
+(NB: If the concept of exporting and symbol tables in Perl is new to you, read C<perldoc -f use>. The documentation for L<Exporter> may also help you as we go along, but bear in mind that not all Perl modules - including Dancer2 - make use of Exporter.)
```

Comment:
Ugh. I wrote a bunch of text and accidentally pressed Ctrl+W. Okay, starting over...

I understand why you're writing it this way, but Dancer is different than modules in generals, since it provides a DSL, in which many keywords make sense. I would prefer a phrasing which doesn't make the author imagine there's only one way. For example:

As a web framework, Dancer2 provides you with many keywords in which you write your web code. Usually, this provides no problem, but in some cases Dancer2 might provide you with a function that exists in another module you wish to use. For example, Dancer2 gives you `any` in order to write endpoints that work on every (or a subset) of the HTTP verbs, while `List::Util` gives you `any` as well.